SIL.Core.Desktop/Properties/Settings.Designer.cs

```diff
@@ -30,9 +30,11 @@ internal sealed partial class Settings : global::System.Configuration.Applicatio
   <string>.mp3</string>
   <string>.wav</string>
   <string>.wma</string>
-  <string>.acc</string>
+  <string>.aac</string>
```

Comment:
Add webm container format for audio

```suggestion
  <string>.aac</string>
    <string>.webm</string>
```

Reply:
discarding this suggestion per comments below